<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serviceWidget</class>
 <widget class="QDialog" name="serviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adjustmentButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>570</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>Юстировка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>Рабочий режим</string>
   </property>
  </widget>
  <widget class="QPushButton" name="soundButton">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>500</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>Оповещение</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>385</y>
     <width>481</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="applyBurstLength">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>290</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
  <widget class="QLabel" name="ammunitionValue">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Патронов</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ammunitionInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="shortUnitLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>273</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выстрелов</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="longUnitLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>335</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выстрелов</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="shortBurstLengthInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>263</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="longBurstLengthInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>325</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="shortLabel">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>275</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Длина короткой очереди</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="longLabel">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>335</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Длина длиной очереди</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyAmmunition">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>481</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="singleFireLabel">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>405</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Длительность выстрела</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="singleShotTimeInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>395</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="singleFireUnitLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>405</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>мс</string>
   </property>
  </widget>
  <widget class="QLabel" name="burstlengthTitleLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Установка длины очереди</string>
   </property>
  </widget>
  <widget class="QLabel" name="ammunitionTitleLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Боезапас</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>220</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>220</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="compressionTitle">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сжатие</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="h264Button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>H264</string>
   </property>
  </widget>
  <widget class="QLabel" name="compressionValueLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mjpegButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MJPEG</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>50</y>
     <width>3</width>
     <height>131</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>240</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>240</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="formatVideoTitleLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Формат видео</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
